es-core/src/resources/Font.cpp

```diff
@@ -416,6 +416,30 @@ void Font::renderTextCache(TextCache* cache)
 	}
 }
 
+Vector2f Font::sizeCodePoint(unsigned int character, float lineSpacing)
+{
+	float lineWidth = 0.0f;
+	float highestWidth = 0.0f;
```

Comment:
What's this for?

Reply:
`highestWidth` is a remnant from `sizeText` (on which `sizeCodepoint` was based). I forgot to delete it.
I'll delete it and update the branch.